management/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
 
     api project(":router")
     api project(":runtime")
+    api project(":http-server")
```

Comment:
This should not be an api dependency. You can have a compileOnly dependency on the `http` project and ensure any http specific beans require the presence of an embeddedserver bean

Reply:
Shouldn't the compileOnly dependency be on `http-server` (rather than `http`)?

The controller uses `HttpHostResolver` which is defined in `http-server`, and I don't see that `http` has a direct (or transitive) dependency on this submodule. Depending on `http` makes the build fail, I think because it does not contain such interface

Reply:
Yep you're right

Reply:
Great, thanks for the response, now it depends as compileOnly on `http-server`